vrk-kpa/suomidigi
# SUOPA-136 Header

To test:
- `make fresh shell`
- `drush cr`
- Go to the site and see that there is header. 
   - Unfortunately the test content is in English, because there is no language configurations in test-environment atm. Will fix them after this PR has been merged.
